Create SECURITY.md (#472)
rupali-codes/LinksHub: SECURITY.md

## Roles and Responsibilities

The project maintainers are responsible for handling vulnerability reports and making decisions about how to address them. The project maintainers will work with contributors and external website owners to resolve vulnerabilities as quickly and responsibly as possible.

## Response Timeline

The project will aim to resolve critical vulnerabilities within 30 days and non-critical vulnerabilities within 90 days. The project may extend these timelines if additional time is needed to address the vulnerability.

##  Secure Coding Practices

The project will provide guidance on secure coding practices for contributors, including guidelines for input validation, authentication, authorization, and data protection
## Regular Review and Update

The security policy will be regularly reviewed and updated to ensure that it remains effective and relevant. The project maintainers will evaluate the vulnerability disclosure process, update secure coding guidelines, and revise the response timeline as needed.
## Disclosure Policy

> The project will follow a coordinated disclosure policy, which means that vulnerabilities will be disclosed publicly only after they have been remediated. The project may work with external website owners to coordinate disclosure of vulnerabilities that affect their websites.

Author: committed all changes

## Legal Disclaimer

> The security policy includes a legal disclaimer that limits the liability of the project maintainers and contributors for any security vulnerabilities or incidents that occur as a result of using the [Linkshub](https://linkshub.vercel.app/) website or any of the linked websites.

Reviewer: ```suggestion
The security policy includes a legal disclaimer that limits the liability of the project maintainers and contributors for any security vulnerabilities or incidents that occur as a result of using[Linkshub's](https://linkshub.vercel.app/) website or any of the linked sources.
```

## Contact Information

> If you have any questions or concerns about the security policy or any security vulnerabilities in the project, please contact us at rupali7487@gmail.com.

Reviewer 2: ```suggestion
If you have any questions or concerns about the security policy or any security vulnerabilities in the project, please contact us at _linkshub.opensource@gmail.com_.
```

> By implementing this security policy, we aim to ensure that vulnerabilities are addressed in a timely and responsible manner, and that users and contributors can use the [Linkshub](https://linkshub.vercel.app/) website and linked websites safely and securely.

Reviewer: ```suggestion
By implementing this security policy, we aim to ensure that vulnerabilities are addressed in a timely manner, and that users and contributors can use [Linkshub](https://linkshub.vercel.app/) and its linked sources safely and securely.
```
